### ISSUE TITLE
Carbon dioxide sensor alarm on A9 and A9 mini

### ISSUE THREAD

**Issue by xszuflax** [tomekkleszcz/homebridge-electrolux-devices#26]
**Describe The Bug:**
The CO2 alarm is triggered on A9 and A9 mini. When the window is opened the CO2 sensor detects 1500 ppm with the closed window it shows 3500 ppm. I'm pretty sure that my home is well-ventilated. I do not see these readings in an Electrolux app.

Is the sensor bad and shows wrong values? or is the plugin showing the wrong reading on A9 air purifiers? 

**To Reproduce:**
N/A

**Expected behaviour:**

From what I read online the correct or in other words healthy value is between 500 - 1500 ppm. I assume that having A9 next to the opened window I should be in that range :-) ?  ATM I set CO2 alarm at 4000ppm just to clear the alarms but it would be nice to have a real reading.

**Logs:**
N/A

Show the Homebridge logs here, and remove any sensitive information.
N/A

**Plugin Config:**
N/A

**Screenshots:**
N/A

**Environment:**

* **Plugin Version**: 0.0.4-beta1
* **Homebridge Version**: 1.6.1
* **Node.js Version**: 18.16.0
* **NPM Version**:  9.5.1
* **Operating System**: Raspbian 11.7 / hb-service 4.50.6

**Comment by xszuflax** [comment]
Just to give you some more background. I live in a town next to Wrocław Poland. The city itself does have pollution issues but I would assume that the countryside should be good with CO2 levels...

Honestly, I try to determine if this sensor gives some deterministic values cuz maybe it the surroundings.

**Comment by xszuflax** [comment]
I tried to find any correlation with the Electrolux app and it looks like [TVOC](https://learn.kaiterra.com/en/resources/understanding-tvoc-volatile-organic-compounds) is in that range and it kinda resembles the CO2 reading reported by the plugin:

![IMG_0037](https://github.com/tomekkleszcz/homebridge-electrolux-devices/assets/10434553/6961c3ce-a598-470d-9132-8b9eae713dd0)

**Comment by pandizajner** [comment]
I'm having the same issue. Even though I change the "Carbon dioxide sensor alarm value" to values like 2000, I continuously get alerts when readings are 1050-1100ppm (not even sure if they're proper). 

Ideally would be to get chance of disabling the carbon dioxide sensor or fix it. I see some PR from @mafredri that was around this topic. @tomekkleszcz is it possible to take a look at this? Thank you!

**Comment by hevalito** [comment]
I have the same issue on my end. I also searched for a solution to disable the sensor as I think the A9 doesn't even have a Carbon Dioxide Sensor Hardware, right?

**Comment by mafredri** [comment]
@hevalito correct, it doesn't have a CO2 sensor, but it does have an _estimated_ CO2 value, but if you inspect the behavior, you can see it's highly correlated with TVOC.

For the time being, and considering this project hasn't seen action in 5+ months, I've been thinking about starting a semi-official fork for this project. I'd likely do some heavy refactoring though, which will make it harder to reintegrate the changes back into this project, if @tomekkleszcz finds more time for it. (Ultimately, I'm not interested in long-term feature development beyond fixing my own use-cases, and maintenance.)

**Comment by xszuflax** [comment]
> For the time being, and considering this project hasn't seen action in 5+ months, I've been thinking about starting a semi-official fork for this project.

If you need a tester I'm up for it :-)

**Comment by tomekkleszcz** [comment]
@xszuflax The carbon dioxide level alerts should be resolved in the new v0.0.5 version. Please let me know if the new release has fixed your issue. Otherwise, I'll reopen this issue.

**Comment by xszuflax** [comment]
Hello @tomekkleszcz ,

welcome back :-) I can confirm that this is fixed for now and setting alert level to 5000 does not trigger alarms in the Home.app.

The more general question though is why it is there at least for A9 units since they do not have CO2 sensor but TVOC. Maybe it would be good to have a toggle to switch it off if one does not feel like having it.

Any way many thanks for your work!
